web/impact/impact/v1/views/office_hours_calendar_view.py

```diff
@@ -0,0 +1,131 @@
+from datetime import (
+    datetime,
+    timedelta,
+)
+
+from pytz import utc
+
+from rest_framework.response import Response
+
+from django.contrib.auth import get_user_model
+from django.db.models import (
+    BooleanField,
+    Case,
+    Count,
+    F,
+    When,
+    Value,
+)
+
+from . import ImpactView
+from accelerator.models import MentorProgramOfficeHour
+User = get_user_model()
+
+ISO_8601_DATE_FORMAT = "%Y-%m-%d"
+ONE_DAY = timedelta(1)
+ONE_WEEK = timedelta(8)
+
+
+class OfficeHoursCalendarView(ImpactView):
+    permission_classes = []
+    view_name = "office_hours_calendar_view"
```

Comment:
Non-mentor users should get an appropriate message when they hit this end point

Reply:
I think this is a **non-blocker** since this endpoint is meant for not only mentors but also staff and finalists and this requirement is not in the definition of done.

Reply:
Agree with @iankigen 's point.